modules/gatk_rnaseq/1.0/envs/gatk_rnaseq.yaml

```diff
@@ -0,0 +1,352 @@
+name: bioinformatics
```

Comment:
Is this environment just GATK or there is something else besides it that is required for this module to run? There is a GATK environment `lcr-modules/envs/gatk/gatk-4.1.8.1.yaml`, maybe it can be reused here? I see this one contains tools like, for example, bedtools, vcf2maf, tmux, and a lot of perl dependencies, but are they used in the module?

Reply:
Hm that is a good question. I pulled it from a working environment. I'll test to see if it'll work with the gatk module

Reply:
Yes it worked with gatk module. I will just use the symlinked version for both gatk and bcftools